runtime/doc/editing.txt

```diff
@@ -1669,11 +1669,11 @@ the |:trust| command or the |vim.secure.read()| function.
 			`:trust` adds the file associated with the current
```

Comment:
```suggestion
			:trust adds the file associated with the current
```

Reply:
Done in a87789fd1a4710209d3558f6314ea40713d43ddf